### ISSUE TITLE
fullPage = true compatibility

### ISSUE THREAD

**Issue by cardillator** [w8tcha/CKEditor-TextSelection-Plugin#30]
Hi,
This version is still limited to config.fullPage = false, isn't it? Is it expected to be complatible with the fullPage = true configuration?
Thanks!

**Comment by w8tcha** [comment]
Yes the plug in is automatically disabled when you enable fullpage mode.
